test/runtests.jl

```diff
@@ -1,9 +1,19 @@
-using FTPServer
 using FTPClient
+using FTPServer
+using FTPServer: Server, serve, username, password, hostname, port, HOMEDIR
 using Memento
 using Memento.TestUtils: @test_log
 using Test
 
+
+function uri(server::Server; ftp_type::String="ftp")
+    string(
+        "$ftp_type://$(username(server)):$(password(server))",
+        "@$(hostname(server)):$(port(server))",
+    )
+end
```

Comment:
I'm ok with putting this in the FTPServer package

Reply:
Otherwise I would just use the `opts` technique used by the other tests in this package

Reply:
ah oh yeah. I think there was an issue to add this to the package itself.